flocker/control/httpapi.py

```diff
@@ -528,6 +552,29 @@ def datasets_from_deployment(deployment):
                 )
 
 
+def container_configuration_response(application, node):
+    """
+    Return a container dict  which confirms to
+    ``/v1/endpoints.json#/definitions/configuration_container``
+
+    :param Application application: An ``Application`` instance.
+    :param unicode node: The host on which this application is running.
+    :return: A ``dict`` containing the container configuration.
+    """
+    image = u'{0}:{1}'.format(
```

Comment:
I assume there's a `DockerImage.to_string`. If there isn't this functionality should be added there.